cmd/minishift/cmd/openshift/config.go

```diff
@@ -27,8 +27,8 @@ const (
 
 var configCmd = &cobra.Command{
 	Use:   "config SUBCOMMAND [flags]",
-	Short: "Displays or patches OpenShift configuration.",
-	Long:  "Displays or patches OpenShift master or node configuration. Patches are supplied in JSON format.",
+	Short: "Displays or updates the OpenShift configuration.",
```

Comment:
We already use "update" in other config commands so I'm aligning with existing (agreed upon) terminology.